src/lib/stores/AccessState.ts

```diff
@@ -0,0 +1,14 @@
+import { readable } from 'svelte/store';
+import { page } from '$app/state';
+import { features } from '$lib/configuration';
+import { isUserLoggedIn } from '$lib/stores/User';
+import { browser } from '$app/environment';
+
+export const useAuth = readable(!features.login.open || isUserLoggedIn());
```

Comment:
These wont update when the user store value changes - maybe make them derived values?